### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard: Trigger Parameter Hints (Ctrl+Shift+') shortcut command is not working.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52680]
Environment Details:
VSCode Version : 1.24.0

Additional Details:
MAS Violated: MAS2.1.1

Repro Steps:
1)Launch VS Code.
2)Open Keyboard Shortcuts.
3)Search the Keyboard shortcut for Trigger Parameter Hints. 

Actual:
Trigger Parameter Hints (Ctrl+Shift+') shortcut command is not working when used on Editor. As it says, it should trigger hints in the editor when user uses this command but it does not show any hints.

Expected:
Trigger Parameter Hints (Ctrl+Shift+') shortcut command should work.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for reference:
![screenshot 65](https://user-images.githubusercontent.com/39204909/41893000-dd18801c-7937-11e8-85c1-7334718c161a.png)

[A11y VSCode Keyboard Shortcuts Trigger Parameter Hints.pptx](https://github.com/Microsoft/vscode/files/2127623/A11y.VSCode.Keyboard.Shortcuts.Trigger.Parameter.Hints.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
Could be #52681.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Added screenshot in the "Attachment for Reference" section.
